libs/aml/src/docs.test.ts

```
@@ -68,6 +64,24 @@ describe('AML docs', () => {
     })
 })
 
+function listFiles(dir: string): string[] {
+    return fs.readdirSync(dir, {recursive: true}).map(path => pathJoin(dir, path as string))
+}
+
+function getAmlSnippets(path: string, content: string): {snippet: string, version?: string}[] {
+    if (path.indexOf('/demos/') !== -1) return [{snippet: content}] // files in demos are fully AML
+    if (path.endsWith('.aml')) return [{snippet: content}] // aml file
+    if (path.endsWith('.md')) return (content.match(/```aml(v1)?[^]*?```/gm) || []).map(snippet => ({
```

Biome

[error] The regular expression includes this negated empty character class.

Negated empty character classes match anything.
If you want to match against [, escape it \[.
Otherwise, remove the character class or fill it.

(lint/correctness/noEmptyCharacterClassInRegex)